tests/scripts/initialize_database.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+docker ps -a
+docker images
+
+docker build --tag elastic_store .
+docker run \
+    --network=bridge \
```

Comment:
Why do you specify this ? Either do not specify a network (it will use the default docker bridge), or use network host

Reply:
My bad, I need this to test in local (host doesn't work) and i forgot to revert it